rootfs/rootfs/etc/rc.d/xedaemon

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ -x /etc/init.d/xe-linux-distribution ]; then
+	/etc/init.d/xe-linux-distribution start
```

Comment:
Thank you for the remind, I added a tips to this file. https://github.com/phusl/boot2docker/commit/ad6c85f703658bbbe1998de889821c467307dd52
:+1: